examples/with-cli/openapi.generated.yaml

```diff
@@ -106,18 +106,14 @@ paths:
                                 type: string
 components:
     schemas:
-        github.com_gin-gonic_gin.H:
```

Comment:
Why did this get removed? (Side note - we should add validation to these examples and pipelines)

Reply:
Agreed, It would also remind us to update them it we add functionality.